pkg/cloud/openstack/cluster/clusteractuator.go

```diff
@@ -0,0 +1,91 @@
+package openstack
```

Comment:
package cluster?
maybe renaming clusteractuator.go to actuator.go.

Reply:
darn it, I thought I had done it.